moped-editor/src/components/GridTable/Search.js

```diff
@@ -212,17 +217,19 @@ const Search = ({
                           Search
                         </Button>
                       </Hidden>
-                      <FormGroup sx={{ display: "inline" }}>
-                        <FormControlLabel
-                          control={
-                            <Switch
-                              checked={showMapView}
-                              onChange={handleMapToggle}
-                            />
-                          }
-                          label="Map"
-                        />
-                      </FormGroup>
+                      <ToggleButtonGroup
+                        className={classes.toggleButtonGroup}
+                        value={showMapView}
+                        exclusive
+                        onChange={handleMapToggle}
+                      >
+                        <ToggleButton aria-label="list view">
```

Comment:
@tillyw The toggle buttons look so much better than the switch! ✨ Could you pass a value prop to these ToggleButtons so that it will highlight which one is active? 🙏

<img width="167" height="113" alt="Image" src="https://github.com/user-attachments/assets/3c8dd6c8-aab5-4e37-a948-264672f3efc5" />